### PR TITLE
Feature/api

### PR DESCRIPTION
**Pull request by SeoyoungC**
Associated with, and closes, [Issue #14](https://github.com/Edds390/solarnetwork-aggregated-dashboard/issues/14)

Implemented API call to pull live data, and to update the graph. 
___
**Changes:**
- Added api.js to hold queries. Currently only has one query (getNodeUsageData), which is an async method to fetch results from a live server. Async/Await is used opposed to ```.then()``` for reading clarity, as discussed [here](https://hackernoon.com/6-reasons-why-javascripts-async-await-blows-promises-away-tutorial-c7ec10518dd9?gi=2b064017bfc8).

- Added method to call the above API to pull data into Dashboard Panel. Promises used to fetch multiple nodes data in parallel, and ```Promise.all()``` used to wait for all fetches to complete. The results of the nodes are concatenated into one array (as the dygraph needs a single array of data), and passed to the dygraph.

- Removed some redundant dummy data imports and references
____
**Known Issues**
- Currently does not handle node IDs that do not exist, [issue #20](https://github.com/Edds390/solarnetwork-aggregated-dashboard/issues/20)
- Date range is currently hardcoded - need to pass these values in, waiting on [issue #9](https://github.com/Edds390/solarnetwork-aggregated-dashboard/issues/9)

### CODE REVIEW

**Review by Edds390 — COMMENTED** [review]
Looks good. Make a new issue for handling those nodes that doesn't exist.

**Review by Edds390 — APPROVED** [review]

File: src/api/api.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
+const HOST = 'https://data.solarnetwork.net';
+
+/**
+ * This method returns usage data for a single node (nodeId) between a set time (start and end date).
+ * */
+const getNodeUsageData = async (nodeId, startDate, endDate) => {
+  const query = `https://data.solarnetwork.net/solarquery/api/v1/pub/datum/list?nodeId=${nodeId}&startDate=${startDate}T0%3A19&endDate=${endDate}T0%3A19&aggregate=Hour`;
```
Comment (elisalter) [inline src/api/api.js]: probably best to split this query over multiple lines, easier to read and update parameters :)

File: src/components/DashboardPanel/DashboardPanel.js [hunk 2]
```diff
@@ -28,10 +26,32 @@ export default class DashboardPanel extends Component {
...
+    this.pullData();
+  }
+
+  /**
+   * Pulls data for all nodes in selectedNodes. Async method, to wait for fetch to finish.
+   */
+  async pullData() {
+    const promiseList = [];
+    this.props.selectedNodes.forEach((node) => {
+      promiseList.push(getNodeUsageData(node, this.state.startDate, this.state.endDate));
```
Comment (elisalter) [inline src/components/DashboardPanel/DashboardPanel.js]: could use object deconstruction here.

`const { startDate, endDate } = this.state;`

**Review by elisalter — CHANGES_REQUESTED** [review]
To improve readability split the query into multi line and use object deconstruction :) see one line comments.

**Review by elisalter — APPROVED** [review]

**Pull request merged**